docs/sdks/server-sdks/ruby.md

```diff
@@ -16,6 +16,12 @@ Install the SDK with gem:
 gem install eppo-server-sdk
 ```
 
+or add to you `Gemfile`:
+
+```
+gem 'eppo-server-sdk, '~> 0.3.0'
```

Comment:
I wonder if we should add a note to use the latest version as this will get out of date

Reply:
I think we should keep this in sync with the latest version as we publish new releases

Reply:
could also remove the version specification, so Bundler will pick the latest, but it's good practice to pin the version